CHANGELOG.md

```diff
@@ -2797,6 +2797,7 @@ BREAKING CHANGES
 
 DEPRECATED
 * [cli] Deprecated `--name` flag in commands that send txs, in favor of `--from`
+* [upgrade] Deprecated time based upgrades.
```

Comment:
this section is for a really old version of the SDK.

As per the comments on the top of this doc, we should add a new `## Deprecated` inside `Unreleased`, and add this line there.